sentinel-2/ozone_column_over_snow/script.js

```diff
@@ -0,0 +1,101 @@
+function setup () {
```

Comment:
According to Javascript style guides there shouldn't be any space between `setup` and `()`. Same applies to all other functions.